charts/pega/README.md

```diff
@@ -480,6 +480,17 @@ Parameter           | Description    | Default value
 `hpa.enableCpuTarget` | Set to true if you want to enable scaling based on CPU utilization or false if you want to disable it | true
 `hpa.enableMemoryTarget` | Set to true if you want to enable scaling based on memory utilization or false if you want to disable it (Pega recommends leaving this disabled) | false
 
+### Setting a Pod Disruption Budget (PDB)
+You can configure a Kubernetes `PodDisruptionBudget` on your tier by setting `pdb.enabled` to `true`.  For more details, see the Kubernetes 
+[Pod Disruption Budgets documentation](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets).
+
+The following parameters are mutually exclusive.  You must specify either `pdb.minAvailable` or `pdb.maxUnavailable` but not both.
+
+Parameter             | Description    | Default value
+---                   | ---       | ---
+`pdb.minAvailable`    | The minimum number of pods in the tier that must remain available at all times  | `1`
+`pdb.maxUnavailable`  | The maximum number of pods in the tier that can be unavailable at one time      | Disabled when `pdb.minAvailable` is specified
```

Comment:
this isn't a value, it's information about usage. Should this be in the description? also, make it active>>Disabled when you specify a `pdb.minAvailable` value.

Should this mention percentage if the commented out values are percentages?